### DISCUSSION TITLE
Create a subset for a defined orientation +/- tolerance

### DISCUSSION THREAD

**Discussion by aleslagter** [mtex-toolbox/mtex#2135] [Ask Anything]
Hello! This might be a very simple question, but I am new to matlab and mtex.
I am loading an EBSD map and plotting pole figures, but I would like to be able to create subsets of the data for a certain orientation and with a defined tolerance. 
I have all the orientations for a phase loaded on

o= ebsd('Zr02').orientations;

and can plot for certain planes h using 
plotPDF(o,h)

but I would like to segment the orientations and create subsets, and also to be able to plot the maps of those orientations.

Thank you for any suggestions!

**Comment by zmichels** [comment]
Probably someone has an even better approach than this... but if you can define one-or-more reference orientation/s of interest, you can compute the angular deviation of any other orientations... and then you can plot the EBSD points colored by the angular difference to visualize their spatial distribution in a map.

**Comment by aleslagter** [comment]
Thank you! I found now the following online and I think it works well

o2 = orientation.byEuler(5*degree, 80*degree, 120*degree,ebsd('Zr02').CS);
grains_selected = grains.findByOrientation(o2,20*degree)

Thank you!
